oshmem/mca/atomic/ucx/atomic_ucx_module.c

```diff
@@ -51,6 +51,11 @@ int mca_atomic_ucx_op(shmem_ctx_t ctx,
     status = ucp_atomic_post(ucx_ctx->ucp_peers[pe].ucp_conn,
                              op, value, size, rva,
                              ucx_mkey->rkey);
+
+    if (OPAL_LIKELY(UCS_OK == status)) {
+        mca_spml_ucx_handle_put_op(ucx_ctx, pe);
```

Comment:
maybe rename 'mca_spml_ucx_handle_put_op' to 'mca_spml_ucx_remote_op_posted'
it's not just 'put' now, also atomic